### ISSUE TITLE
libpng warning, global_step and eval_batch_step

### ISSUE THREAD

**Issue by MohieEldinMuhammad** [PaddlePaddle/PaddleOCR#7798]
#### 1. While training i got these 2 warnings  and don't know how to deal with them or why they are appearing: 
    1.1 **libpng warning: iCCP: known incorrect sRGB profile**
    1.2 **libpng warning: iCCP: profile 'ICC Profile': 'RGB ': RGB color space not permitted on grayscale PNG**
#### 2. I'm setting the  ```eval_batch_step = [0,200]``` which means the evaluation will be after every 200 iteration and my ```training set is 3,712 image``` and my ``` test set is 984 image```
    2.1 I'm not understating what is the meaning of iteration, i need to know after how many image the model will be evaluated? 
    2.2 what is the meaning of global_step?
  
  
![image](https://user-images.githubusercontent.com/89604782/193227589-e47be5e3-70d9-4481-a05a-14213142a26c.png)

**Comment by MohieEldinMuhammad** [comment]
Also sometimes it gives me ```eval model:: 100% 8/8 [00:02<00:00,  3.03it/s]``` 
the 8 is the number of images divided by batch size but why it appears occasionally ? 
![Screenshot (2)](https://user-images.githubusercontent.com/89604782/193232024-a8d616a0-91ab-4a9d-b6ad-de47bf4554f2.png)

**Comment by MohieEldinMuhammad** [comment]
Any help here guys ?  @MissPenguin

**Comment by zhangqiqi1228** [comment]
> #### 1. While training i got these 2 warnings and don't know how to deal with them or why they are appearing:
> ```
> 1.1 **libpng warning: iCCP: known incorrect sRGB profile**
> 1.2 **libpng warning: iCCP: profile 'ICC Profile': 'RGB ': RGB color space not permitted on grayscale PNG**
> ```
> 
> #### 2. I'm setting the `eval_batch_step = [0,200]` which means the evaluation will be after every 200 iteration and my `training set is 3,712 image` and my ` test set is 984 image`
> ```
> 2.1 I'm not understating what is the meaning of iteration, i need to know after how many image the model will be evaluated? 
> 2.2 what is the meaning of global_step?
> ```
> 
> ![image](https://user-images.githubusercontent.com/89604782/193227589-e47be5e3-70d9-4481-a05a-14213142a26c.png)

Hi, have you solved the problem 1.1? I'm facing the same problem.
